plant-collect.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+//Create function to initiate connection with database and create query
+
```

Comment:
These explanatory comments before functions are kind of redundant when our functions are named well - we know from the name that `connectDb` connects to our database.

What we can also see in this comment is that it is out of date and wasn't updated when the function it is near changed - this function doesn't create a query any more. This is another hazard of 'helpful' explanatory comments, they quickly get out-of-date with what the code actually does and can become misleading, which is why we don't do them.